js/ascendex.js

```diff
@@ -1703,6 +1704,112 @@ module.exports = class ascendex extends Exchange {
         return await this.v2PrivateAccountGroupGetFuturesPosition (this.extend (request, params));
     }
 
+    parseFundingRate (fundingRate, market = undefined) {
+        //
+        // {
+        //     time: "1640078764560",
+        //     symbol: "BTC-PERP",
+        //     markPrice: "48499.644083351",
+        //     indexPrice: "48530.35",
+        //     openInterest: "252.1125",
+        //     fundingRate: "-0.000631045",
+        //     nextFundingTime: "1640102400000"
+        // }
+        //
+        const marketId = this.safeString (fundingRate, 'symbol');
+        const symbol = this.safeSymbol (marketId, market);
+        const currentTime = this.safeInteger (fundingRate, 'time');
+        const nextFundingRate = this.safeNumber (fundingRate, 'fundingRate');
```

Comment:
I think this should actually be `previousFundingRate` 

@frosty00  ?

Reply:
I think it's nextFundingRate because it's consistently changing if I call the method a few minutes apart